### ISSUE TITLE
Installation wizard - Opsgenie channel configuration test button

### ISSUE THREAD

**Issue by crsfalzon** [SimplyVC/panic#270]
### User Story
As a node operator, I want to be able to verify that the Opsgenie channel I set up during installation is correctly configured before my settings are finalised.

### Description
The scope of this task is **not** limited exclusively to the frontend aspect of a test button for Opsgenie channel setup. This ticket includes the functionality for sending out of a test alert on the channel being set up. - see https://github.com/SimplyVC/panic/issues/47.

### Requirements
- When setting up a Opsgenie channel the user requires a Test button. (When assigning severities to the channel, setting configuration name, API token, enabling/disabling the EU region checkbox). 
- Clicking on the Test button triggers the issue of a test alert to the channel being configured (as is on current installer).
- If the test alert is successfully sent to the channel, show a message reading, **"Test alert sent to _channel-name_"**.
- If the test alert isn't issued, show a message reading, "**Error while sending test alert to _channel-name_"**.
- Use toast messages to display test feedback. Timeout = 5s

### Blocked by
Not dependent on another ticket to be tackled.

### Acceptance criteria
**Scenario**: Node operator is setting up an Opsgenie channel for alerting. 
**Given**: The node operator is adding/updating an Opsgenie channel.
**When**: He clicks on the Test button.
**Then**: His configuration is tested by sending a test alert on the node operator's Opsgenie channel.
**And**: If his configuration is valid.
**Then**: Show a message reading, "Test alert sent to _channel-name_".
**And**: If his configuration is invalid.
**Then**: Show a message reading, "Error while sending test alert to _channel-name_".
